ssl/handshake_server.cc

```diff
@@ -399,6 +399,8 @@ static const SSL_CIPHER *choose_cipher(
 static enum ssl_hs_wait_t do_start_accept(SSL_HANDSHAKE *hs) {
   ssl_do_info_callback(hs->ssl, SSL_CB_HANDSHAKE_START, 1);
   hs->state = state12_read_client_hello;
+  ssl_update_counter(hs->ssl->session_ctx.get(),
+                     hs->ssl->session_ctx->stats.sess_accept, true);
```

Comment:
What are these info callbacks used for? In some places they're called before the counter is updated, and others after. Could these info callbacks use the counter and expect to see (or not see) the up to date counter?